plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -324,6 +324,15 @@ function perflab_install_and_activate_plugin( string $plugin_slug, array &$proce
 		return $plugin_data;
 	}
 
+	// Add recommended plugins (soft dependencies) to the list of plugins installed and activated.
+	$standalone_plugin_data = perflab_get_standalone_plugin_data();
+	if ( isset( $standalone_plugin_data[ $plugin_slug ]['suggest'] ) ) {
+		$plugin_data['requires_plugins'] = array_merge(
+			$plugin_data['requires_plugins'],
+			$standalone_plugin_data[ $plugin_slug ]['suggest']
+		);
+	}
```

Comment:
See my other comment: For those reasons, I think we should simply hard-code here: If `embed-optimizer`, add `optimization-detective` to `requires_plugins`.